docs/apps/components/date.md

```diff
@@ -0,0 +1,234 @@
+|**Label Wrap**| If the label is longer than the allowed width the text will continue on another line|<img src= "/apps/components/img/input_labelwrap1.png">|
+|**Hide Label**| Hides the label on the form|<img src= "/apps/components/img/input_hidelabel.png">|
+|**Text Prompt**| A description to aid the user when completing the field|<img src= "/apps/components/img/input_textprompt.png">|
+|**Display Type**| You can select from various options including **Year**, **Month**, **Date**, **Week**, **Dates**, **Datetime**, **Datetimerange**, **Daterange**, or **Monthrange**|<img src= "/apps/components/img/date_displaytype.png">|
+|**Placeholder**| The short hint is displayed in the input field before the user enters a value|<img src= "/apps/components/img/input_placeholder.png">|
+|**Format**| You can decide the format of your date like **YYYY-MM-DD** or **DD/MM/YY**|<img src= "/apps/components/img/date_format.png">|
+|**Default Value**|Specifies the initial value that's displayed in the date field when the form is loaded|<img src= "/apps/components/img/date_defaultvalue.png">|
+|**Custom Class**| An HTML class attribute which allows further customisation [See Form Attribute > Style Sheets](https://bani-appsection--connexcs-docs.netlify.app/apps/page-builder/#form-attribute)|<img src= "/apps/components/img/input_customclass.png">|
+|**Attribute Action**|Enable **Data Binding** to connect the data to UI. <br>Enable **Read Only** to make the field non-editable. <br>Enable **Disabled** action to make the field unusable. <br> Enable **Text box is editable** to allow users to manually type in a date value instead of only selecting from the date picker. <br>Enable **Show Clear button** action to make the clear button visible|<img src= "/apps/components/img/date_attributeaction.png">
+
+### Validation
+
+Form validation is the process of checking the data entered into a form to ensure that it's valid and complete.
+
+This helps to prevent users from submitting forms with invalid data, which can cause problems for the application that's processing the form.
+
+Form validation can be performed using a variety of methods, including:
+
+| **Name**| **Description**|
+|---------|----------------|
+| **Required**| Single line of text|
+|**Custom Validation Rules**|(rule, value, callback) => {|
+||**rule**: Verification rule, you can view the verification configuration information through this parameter; **rule.field** can get the field identifier of the current verification.|
+||**value**: Value of the current field|
+||**callback**: Callback function (must be called) upon completion of validation; **callback('Error message')/ callback(new Error('Error message'))**. These are two ways to return an error message.|
+
+<img src= "/apps/components/img/cascader_validation.png">
+
+!!! Info
+    1. The **callback()function** is also called to verify success in the custom checkup method.
```

Comment:
Please add a space between callback() and function. like this ... **callback()** function ...

Reply:
done